docs/docs/configuration.md

```diff
@@ -4,4 +4,92 @@ title: Configuration
 
 # Configuration
 
-TODO.
+## Mesosphere DCOS
+
+To configure Jenkins on Mesos for the Mesosphere DCOS, create a new file called
+`options.json` in your working directory. This file will contain options
+specific to your environment, such as the name of the framework (e.g.
+`jenkins-test`, `jenkins-release`, etc), and the path to the NFS share you want
+to save your Jenkins configuration and build data to.
+
+The following example will create a new DCOS service named `jenkins-myteam` and
+use the NFS share located at `/mnt/nfs/jenkins_data`:
+
+```
+{
+    "jenkins": {
+        "framework-name": "jenkins-myteam",
+        "host_volume": "/mnt/nfs/jenkins_data"
+    }
+}
+```
+
+*Tip: the value of `host_volume` is the base path to a share on a NFS server
+or other distributed filesystem. The actual path on-disk for this particular
+example will be `/mnt/nfs/jenkins_data/jenkins-myteam`.*
+
+Assuming your DCOS service options have been saved to the file `options.json`,
+you can install Jenkins with your site-specific configuration by running
+the following command.
+
+```
+$ dcos package install jenkins --options=options.json
+```
+
+Wait a few moments for the instance to become healthy, then access the
+`jenkins-myteam` service via the DCOS UI.
+
+## Marathon Open Source
```

Comment:
You can use the Universe and `dcos` CLI with open source Marathon (you just need to set the `marathon.url` parameter) - it might be easier to omit these instructions and just provide configuration guidelines.